Utilities/CollectBenchmarkSucceeded.wdl

```diff
@@ -0,0 +1,62 @@
+version 1.0
+
+workflow CollectBenchmarkSucceeded {
+    input {
+        String workspace_bucket_name
+        String submission_id
+    }
+
+    call CombineSucceededVCFs {
+        input:
+            ws_name = workspace_bucket_name,
+            sub_id = submission_id
+    }
+
+    output {
+        File combined_summaries = CombineSucceededVCFs.combined_summaries
+    }
+}
+
+
+task CombineSucceededVCFs {
+    input {
+        String ws_name
+        String sub_id
+    }
+
+    command <<<
+        # Keep track of shard number in temp file to handle loss of scope in loop
+        echo 0 > number.txt
+
+        # Double loop allows compatibility with one or multiple workflow sample pairs
+        gsutil ls gs://~{ws_name}/~{sub_id}/FindSamplesAndBenchmark/ | while read -r workflow ; do
+            gsutil ls "${workflow}call-BenchmarkVCF" | while read -r shard ; do
```

Comment:
Suggest storing the list: 
`shard_list=($(gsutil ls "${workflow}call-BenchmarkVCF))` and then using a for loop for each shard in the list.